sql/qaqc/qaqc_app_additions.sql

```diff
@@ -15,3 +15,27 @@ SELECT a.job_number,
 	END) as classa_net_mismatch
 INTO qaqc_app_additions
 FROM FINAL_devdb a;
+
+ALTER TABLE qaqc_app_additions ADD manual_hny_match_check INT;
+WITH manual_hny_match_check AS (
+    SELECT 
+    job_number
+    FROM corr_hny_matches 
+    WHERE 
+    action = 'add' 
+    AND
+    hny_id IN (SELECT hny_id FROM hny_no_match)
```

Comment:
This query comes up empty on my machine

Reply:
yes. As it should because none of the hny records got added to a job number should be in the `hny_no_match` table as the kind of intended results

Reply:
@SashaWeinstein actually I will walk back what I said. Even though the check might be more trivial than what I originally intended, it does still check for whether the hny manual correction insertion step worked. Which kind of good enough for our purposes even though it does not care about dob side of things at all.